demosplan/DemosPlanStatementBundle/Logic/AssessmentExportOptions.php

```diff
@@ -270,10 +270,7 @@ public function get($section)
         return $this->options[$section];
     }
 
-    /**
-     * @return mixed
-     */
-    public function jsonSerialize()
+    public function jsonSerialize(): Collection
```

Comment:
Either I'm missing something or this is very wrong. Why should jsonSerialize return a Collection? Especially when it is used in another method that tries to json encode that collection. I think the naming is throwing me for a loop here but I would prefer the naming to get fixed together with the return type.

Reply:
This is just how the current implementation works. I did not want to change anything (yet)

Reply:
@AlexMenschoid may I merge?

Reply:
ping @AlexMenschoid